draft-ietf-quic-recovery.md

```diff
@@ -1269,19 +1269,40 @@ timers wake up late. Timers set in the past fire immediately.
 Pseudocode for SetLossDetectionTimer follows:
 
 ~~~
-GetEarliestTimeAndSpace(times):
-  time = times[Initial]
+GetLossTimeAndSpace():
+  time = loss_time[Initial]
   space = Initial
   for pn_space in [ Handshake, ApplicationData ]:
-    if (times[pn_space] != 0 &&
-        (time == 0 || times[pn_space] < time) &&
-        # Skip ApplicationData until handshake completion.
-        (pn_space != ApplicationData ||
-          IsHandshakeComplete()):
-      time = times[pn_space];
+    if (time == 0 || loss_time[pn_space] < time):
+      time = loss_time[pn_space];
       space = pn_space
   return time, space
 
+GetPtoTimeAndSpace():
+  duration = smoothed_rtt +
+      max(4 * rttvar, kGranularity) * (2 ^ pto_count)
+  # Arm PTO from now when there are no inflight packets
+  if (no in-flight packets):
+    return now() + duration, Initial
+  timeout = infinite
+  pn_space = Initial
```

Comment:
using space and pn_space is awkward, how about naming this pto_space (and maybe then pto_time for timeout)

Reply:
Sg, changed to pto_timeout and pto_space.